lua/nordic/platforms/init.lua

```diff
@@ -0,0 +1,46 @@
+-- Adapted from [tokyonight](https://github.com/folke/tokyonight.nvim)
+--
+-- Tokyonight is licensed under [Apache License 2.0](https://raw.githubusercontent.com/folke/tokyonight.nvim/refs/heads/main/LICENSE)
+--
+-- Changes:
+--      - Adjusted the format of M.platforms
+--      - Changed variable names (mostly from extra -> platform)
+--      - Simplified code to work better for just one color scheme "flavor"
+local utils = require('nordic.utils')
```

Comment:
Same as above:

```suggestion
local U = require('nordic.utils')
```

Just use lsp rename command and make sure you get all references. I know you know this and I don't want to seem patronizing but I also want to give correct instructions.